src/client/particles.cpp

```diff
@@ -33,33 +33,19 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "client.h"
 #include "settings.h"
 
-/*
-	Utility
-*/
-
-static f32 random_f32(f32 min, f32 max)
-{
-	return rand() / (float)RAND_MAX * (max - min) + min;
-}
-
-static v3f random_v3f(v3f min, v3f max)
-{
-	return v3f(
-		random_f32(min.X, max.X),
-		random_f32(min.Y, max.Y),
-		random_f32(min.Z, max.Z));
-}
-
 /*
 	Particle
 */
 
+ClientParticleTexture::ClientParticleTexture(ParticleTexture p) :
+	ParticleTexture(p) {};
```

Comment:
Generally it's a good idea to move methods without body into the class definition.

Reply:
really, that doesn't cause linking issues? okay, i'll move it